cmd/promtool/unittest.go

```diff
@@ -210,6 +210,12 @@ func (tg *testGroup) test(mint, maxt time.Time, evalInterval time.Duration, grou
 			}
 			for _, g := range groups {
 				g.Eval(suite.Context(), ts)
+				for _, r := range g.Rules() {
+					if r.LastError() != nil {
+						errs = append(errs, errors.Errorf("    rule: %s, time: %s, err: %v",
+							r.Name(), ts.Sub(time.Unix(0, 0)), r.LastError()))
```

Comment:
Why are you subtracting the 0 Unix time?

Reply:
To be consistent with how `promtool` displays its results.

```
Unit Testing:  unit.yml                                                                                               
  FAILED:                                                                                                                                                                                    
    alertname: foobar, time: 169h15m0s, err: query processing would load too many samples into memory in query execution
```

Instead of this without the subtraction:

```
Unit Testing:  unit.yml                                                                                               
  FAILED:                                                                                                                                                                                    
    alertname: foobar, time: 1970-01-07 23:38:00 +0100 CET, err: query processing would load too many samples into memory in query execution
```